src/deprecated.rs

```diff
@@ -78,14 +78,16 @@ impl Epoch {
         self.to_duration_since_j1900()
     }
 
-    #[must_use]
-    #[deprecated(
-        note = "Prefix for this function is now `to_` instead of `as_`.",
-        since = "3.5.0"
-    )]
-    pub fn as_duration_since_j1900_in_time_scale(&self, time_scale: TimeScale) -> Duration {
-        self.to_duration_since_j1900_in_time_scale(time_scale)
-    }
+    /*
```

Comment:
I think that we can remove all of the `deprecated.rs` file since it'll be a new major release, we can clean that up.